cmd/certcontroller.go

```diff
@@ -87,7 +87,13 @@ var certcontrollerCmd = &cobra.Command{
 			setupLog.Error(err, "unable to start manager")
 			os.Exit(1)
 		}
-		crdctrl := crds.New(mgr.GetClient(), mgr.GetScheme(),
+
+		var isLeader bool
+		go func() {
+			<-mgr.Elected()
```

Comment:
This seems to be the only API available to inspect leader status. It's a bit clunky to have to read from that channel and then set the `isLeader=true`. Open for suggestions :thinking: